mk/dependencies/yamllint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+OUTPUT_DIR=$1/bin
+VERSION="1.31.0"
+YAMLLINT="${OUTPUT_DIR}"/yamllint.tar.gz
+VERSION_NAME="v${VERSION}"
+
+if [ -z $(pip --version | grep "command not found") ]; then
```

Comment:
Python and pip are needed to use yamllint. Is it something that should be managed by circleCi directly?
Maybe you also don't want to add Python language. I tried to find if this was possible to use yamllint without Python but couldn't.